[2025-03-31T13:54:03.410+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-31T13:54:03.417+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: check_dag.create_file manual__2025-03-31T08:23:57.308511+00:00 [queued]>
[2025-03-31T13:54:03.421+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: check_dag.create_file manual__2025-03-31T08:23:57.308511+00:00 [queued]>
[2025-03-31T13:54:03.421+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-31T13:54:03.431+0530] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): create_file> on 2025-03-31 08:23:57.308511+00:00
[2025-03-31T13:54:03.433+0530] {standard_task_runner.py:72} INFO - Started process 20672 to run task
[2025-03-31T13:54:03.435+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'check_dag', 'create_file', 'manual__2025-03-31T08:23:57.308511+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/check_dag.py', '--cfg-path', '/tmp/tmp92un11we']
[2025-03-31T13:54:03.436+0530] {standard_task_runner.py:105} INFO - Job 18: Subtask create_file
[2025-03-31T13:54:03.459+0530] {task_command.py:467} INFO - Running <TaskInstance: check_dag.create_file manual__2025-03-31T08:23:57.308511+00:00 [running]> on host Voldemort.
[2025-03-31T13:54:03.499+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='check_dag' AIRFLOW_CTX_TASK_ID='create_file' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T08:23:57.308511+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T08:23:57.308511+00:00'
[2025-03-31T13:54:03.499+0530] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-31T13:54:03.508+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-31T13:54:03.509+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hi there!" >/tmp/dummy']
[2025-03-31T13:54:03.515+0530] {subprocess.py:86} INFO - Output:
[2025-03-31T13:54:03.516+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-31T13:54:03.529+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-31T13:54:03.529+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=check_dag, task_id=create_file, run_id=manual__2025-03-31T08:23:57.308511+00:00, execution_date=20250331T082357, start_date=20250331T082403, end_date=20250331T082403
[2025-03-31T13:54:03.567+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-31T13:54:03.582+0530] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-31T13:54:03.586+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
